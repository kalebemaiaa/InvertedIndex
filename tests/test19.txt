QueryOr(QueryAnd(QueryWord("assessor"), QueryNot(QueryWord("ministro"))), QueryWord("secretário"))
820.raw
3328.raw
5462.raw
3824.raw
5798.raw
477.raw
5972.raw
1457.raw
5496.raw
1673.raw
489.raw
5442.raw
4259.raw
6322.raw
5687.raw
11274.raw
3733.raw
3149.raw
3755.raw
5061.raw
252.raw
11246.raw
857.raw
4020.raw
11502.raw
11125.raw
5996.raw
39.raw
755.raw
3584.raw
1336.raw
4086.raw
3818.raw
11789.raw
11898.raw
2270.raw
6185.raw
3696.raw
3112.raw
11532.raw
4889.raw
136.raw
5159.raw
4165.raw
3460.raw
11342.raw
726.raw
362.raw
314.raw
5847.raw
11080.raw
3199.raw
1165.raw
3642.raw
2918.raw
3224.raw
4019.raw
11096.raw
3982.raw
3265.raw
4833.raw
5834.raw
2123.raw
5206.raw
10829.raw
2774.raw
1725.raw
5904.raw
102.raw
5373.raw
1453.raw
4128.raw
11209.raw
11004.raw
2979.raw
2393.raw
5036.raw
11140.raw
3851.raw
11114.raw
8834.raw
692.raw
1118.raw
8828.raw
3337.raw
10904.raw
1969.raw
287.raw
12015.raw
5548.raw
11679.raw
5524.raw
4600.raw
5250.raw
4634.raw
2954.raw
2614.raw
670.raw
2675.raw
3162.raw
6297.raw
856.raw
7794.raw
2637.raw
2623.raw
11262.raw
4906.raw
4752.raw
6333.raw
3837.raw
11659.raw
4690.raw
6228.raw
2416.raw
1155.raw
12004.raw
3822.raw
2752.raw
3760.raw
11614.raw
524.raw
1489.raw
6039.raw
1023.raw
5453.raw
10836.raw
2051.raw
327.raw
4839.raw
5175.raw
3876.raw
2982.raw
3783.raw
2651.raw
5955.raw
154.raw
3252.raw
2404.raw
11827.raw
1883.raw
1233.raw
8855.raw
6338.raw
5219.raw
5521.raw
11507.raw
10908.raw
5853.raw
474.raw
2946.raw
4402.raw
5526.raw
8846.raw
6400.raw
653.raw
10784.raw
1622.raw
1365.raw
11167.raw
11419.raw
4245.raw
10919.raw
3147.raw
4274.raw
4828.raw
5058.raw
6316.raw
11029.raw
1652.raw
5334.raw
10998.raw
2820.raw
3091.raw
1574.raw
3840.raw
1448.raw
544.raw
6253.raw
5600.raw
11087.raw
11714.raw
3572.raw
1985.raw
1573.raw
59.raw
850.raw
5267.raw
4110.raw
2202.raw
4121.raw
4275.raw
5575.raw
4831.raw
5919.raw
2215.raw
1593.raw
2077.raw
6346.raw
11738.raw
3702.raw
