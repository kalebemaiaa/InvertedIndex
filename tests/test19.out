39
59
102
136
287
327
362
489
670
850
1023
1155
1453
1574
1593
1652
1725
2051
2393
2614
2637
2675
2979
3147
3149
3265
3337
3460
3733
3822
3840
3876
3982
4402
4752
4906
5175
5219
5373
5462
5904
6228
10784
10836
10904
11004
11029
11096
11246
11342
11419
11502
11532
11659
11714
11738
11898
